pkgs/tools/security/nwaku/default.nix

```diff
@@ -0,0 +1,89 @@
+{
+  stdenv, fetchurl, fetchgit, lib, nim, which, writeScriptBin,
+  # Options: wakunode1, wakunode2, wakubridge
+  makeTargets ? [ "wakunode2" ],
+  # WARNING: CPU optmizations that make binary not portable.
+  nativeBuild ? false
+}:
+
+stdenv.mkDerivation rec {
+  pname = "nwaku";
+  version = "v0.15.0";
+  commit = "e12b7cb4";
+  name = "${pname}-${version}-${commit}";
```

Comment:
```suggestion
  version = "0.15.0";
```

Reply:
We need the commit. We need it for the `fakeGit` script and I might as well use it in the `name`.

Reply:
I removed the `v` prefix from `version` tho.